package io.soboro.supreme.core.model.user.exception

class InvalidEmailException : RuntimeException("invalid email has been occurred")
